dist/tools/jlink/jlink.sh

```diff
@@ -86,9 +84,6 @@ test_config() {
         echo "Error: No target device defined in JLINK_DEVICE env var"
         exit 1
     fi
-    if [ -z "${JLINK_FLASH_ADDR}" ]; then
-        JLINK_FLASH_ADDR=${_FLASH_ADDR}
-    fi
```

Comment:
I would prefer that we still check, if `FLASH_ADDR` is set before proceeding, as this script could be called from outside the RIOT makesystem... So a simple `if [ -z "${FLASH_ADDR}" ]; then` -> echo error message should do...